index.js

```diff
@@ -23,9 +23,7 @@ module.exports = function(app) {
         app.use(express.static(path.join(__dirname, 'app')));
 
         viewsDirectory += '/app/views';
-    }
-
-    if ('production' === env) {
+    } else {
```

Comment:
`grunt serve` defaults to serve in `dev`. Judging by the Grunt file, I guess you could use `grunt serve:dist` to do some equivalent of `grunt build && NODE_ENV=production npm start`. Maybe I'll just replace that with a `prestart` script.
`npm start` used to start the app without any environment, resulting in broken layouts due to the implementation this changeset corrects.

As for needing the two modes, this could benefit from a longer investigation, which I kept away from this PR. The current behaviour is already pretty much limited to adding `live-reload`, but there's some work to be done in order to understand how disabled exactly is `prerender`, as well as understanding routes precedency rules.

I don't think this is the place and time to handle this topic. This changeset already makes it clear everything should default to `prod` environment, making `dev` an opt-in. Let's wait until we need to update the rules.